modules/grandpa/src/lib.rs

```diff
@@ -315,10 +295,10 @@ pub mod pallet {
 	pub(super) type ImportedHashesPointer<T: Config<I>, I: 'static = ()> =
 		StorageValue<_, u32, ValueQuery>;
 
-	/// Headers which have been imported into the pallet.
+	/// Relevant fields of imported headers.
 	#[pallet::storage]
 	pub type ImportedHeaders<T: Config<I>, I: 'static = ()> =
-		StorageMap<_, Identity, BridgedBlockHash<T, I>, StoredBridgedHeader<T, I>>;
+		StorageMap<_, Identity, BridgedBlockHash<T, I>, BridgedStoredHeaderData<T, I>>;
```

Comment:
@svyatonik 
I guess, this is not backwards compatible, right?
and there is no migration.

so, for live deployed Rococo/Wococo BridgeHubs, next time I will update runtime which contains this commit, I should clean database and/or reonboard parachains?

It is not a problem, I just want to know.

Reply:
Yes - it isn't backwards-compatible, but iiuc the upgrade will be smooth - headers import is not touching any previous headers, only current authorities set. And we are not touching set here. So the issue will only be with verifying storage proofs using old headers, which we are not doing currently. We'll leave some garbage in Rococo/Wococo runtime storage though. I'm not sure if it is critical, since they're testnets.

Reply:
ok, cool, thank you, so just decoding will fail for old stored headers, this is no problem,
yes, we will have some garbage, but anyway later or sooner, I am sure, we will do clean reonboard :)